### PR TITLE
Define `elector`.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

**Review by dmitrizagidulin — APPROVED** [review]
👍

**Review by gannan08 — APPROVED** [review]
LGTM

File: contexts/veres-one-v1.jsonld [hunk 1]
```diff
@@ -77,6 +77,7 @@
     "digestAlgorithm": "sec:digestAlgorithm",
     "digestValue": "sec:digestValue",
     "domain": "sec:domain",
+    "elector": "wl:elector",
```
Comment (dlongley) [inline contexts/veres-one-v1.jsonld]: Should have `"@type": "@id"` just like the web ledger context, right?

**Review by dlongley — CHANGES_REQUESTED** [review]

**Comment by mattcollier** [comment]
@dlongley ready for another pass.

**Review by dlongley — APPROVED** [review]

**Pull request merged**